src/Makefile.test.include

```diff
@@ -112,6 +112,7 @@ BITCOIN_TESTS =\
   test/net_peer_eviction_tests.cpp \
   test/net_tests.cpp \
   test/netbase_tests.cpp \
+  test/pool_tests.cpp \
```

Comment:
Noticed while running `make check`: the following minor change in the first commit  2bbe0f8eeae274fc30d924c6762cbb7a392f3b32 will run the tests in the expected order.

```diff
   test/net_tests.cpp \
   test/netbase_tests.cpp \
-  test/pool_tests.cpp \
   test/orphanage_tests.cpp \
   test/pmt_tests.cpp \
   test/policy_fee_tests.cpp \
   test/policyestimator_tests.cpp \
+  test/pool_tests.cpp \
   test/pow_tests.cpp \
```

currently

```
Running tests: net_peer_eviction_tests from test/net_peer_eviction_tests.cpp
Running tests: net_tests from test/net_tests.cpp
Running tests: netbase_tests from test/netbase_tests.cpp
Running tests: pool_tests from test/pool_tests.cpp
Running tests: orphanage_tests from test/orphanage_tests.cpp
Running tests: pmt_tests from test/pmt_tests.cpp
Running tests: policy_fee_tests from test/policy_fee_tests.cpp
```

after

```
Running tests: pmt_tests from test/pmt_tests.cpp
Running tests: policy_fee_tests from test/policy_fee_tests.cpp
Running tests: policyestimator_tests from test/policyestimator_tests.cpp
Running tests: pool_tests from test/pool_tests.cpp
Running tests: pow_tests from test/pow_tests.cpp
Running tests: prevector_tests from test/prevector_tests.cpp
Running tests: raii_event_tests from test/raii_event_tests.cpp
```

Reply:
Interesting that the order here has any effect on the order of tests, I would have expected that the test suite orders alphabetically. I'll add this to my next update

Reply:
done in e7158613dcafd0065a94b03c8013ee4ced8ec3e3